packages/genesys-spark-components/src/components/beta/gux-popover-beta/gux-popover.scss

```diff
@@ -0,0 +1,73 @@
+/* This is necessary to override the position of the popover on the page. By default, an HTML popover will
+   render in the middle of the screen, which we do not want */
+:popover-open {
+  position: absolute;
+  inset: unset;
+}
+
+.gux-popover-wrapper {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: var(--gse-semantic-zIndex-popover);
```

Comment:
It looks by using the native api it adds the popover to the top layer. You can see this by scrolling the container in the example file the `popover` will appear over all the elements. Whereas in the `stable` gux-popover it utilises the `z-index` we set. I am not sure if we need this property anymore but we can maybe discuss in collab time.